Library/Homebrew/formula_auditor.rb

```diff
@@ -495,7 +495,7 @@ def audit_specs
         end
 
         next if spec.patches.empty?
-        next unless @new_formula
+        next unless @new_formula && @core_tap
```

Comment:
```suggestion
        next if !@new_formula && !@core_tap
```

Reply:
```suggestion
        next if !@new_formula || !@core_tap
```

Right?

Reply:
Yup but that's why I hate `unless &&`: it's not trivial (for me at least) to figure out how to "flip" it 😁